apps/root/src/pages/home/components/earn-positions-dashboard/index.tsx

```diff
@@ -0,0 +1,102 @@
+  const { assetsTotalValue, totalAssetValue } = useNetWorth({ walletSelector: selectedWalletOption });
+  const { userStrategies, hasFetchedUserStrategies } = useEarnPositions();
+  const showBalances = useShowBalances();
+
+  const filteredPositions = React.useMemo(
+    () =>
+      userStrategies.filter(
+        (strategy) => selectedWalletOption === ALL_WALLETS || strategy.owner === selectedWalletOption
+      ),
+    [userStrategies, selectedWalletOption]
+  );
+
+  const filteredPositionsLenght = filteredPositions.length;
+
+  if (!userStrategies.length) {
+    return null;
+  }
+
+  return (
+    <Grid item xs={12} display="flex">
+      <ContainerBox flex="1">
+        <WidgetFrame
+          assetValue={assetsTotalValue.earn}
+          title={<FormattedMessage defaultMessage="Earn" description="home.earn.dashboard.title.earn" />}
+          subtitle={
+            hasFetchedUserStrategies &&
+            showBalances &&
+            (filteredPositionsLenght === 1 ? (
+              <FormattedMessage
+                defaultMessage="1 active vault"
+                description="home.earn.dashboard.title.vaults.singular"
+              />
+            ) : (
+              <FormattedMessage
+                defaultMessage="{vaults} active vaults"
+                description="home.earn.dashboard.title.vaults.plural"
+                values={{
+                  vaults: filteredPositionsLenght,
+                }}
+              />
+            ))
+          }
+          isLoading={!hasFetchedUserStrategies}
+          collapsable
+          widgetId="Earn Dashboard"
+          totalValue={totalAssetValue}
+          showPercentage
+          Icon={MoneyAddIcon}
+        >
+          {!!filteredPositionsLenght ? (
+            <Grid container justifyContent="space-between" rowSpacing={4}>
+              <Grid item xs={12} md="auto" display="flex" flexDirection="column" gap={3}>
+                <Typography variant="bodyBold" color={({ palette }) => colors[palette.mode].typography.typo1}>
+                  <FormattedMessage defaultMessage="Total Value" description="home.earn.dashboard.title.total-value" />
+                </Typography>
+                <FinancialOverview
+                  userPositions={filteredPositions}
+                  size="small"
+                  isLoading={!hasFetchedUserStrategies}
+                />
+              </Grid>
+              <Grid item xs={12} md="auto" display="flex" flexDirection="column" gap={3}>
+                <Typography variant="bodyBold" color={({ palette }) => colors[palette.mode].typography.typo1}>
+                  <FormattedMessage
+                    defaultMessage="Expected returns"
+                    description="home.earn.dashboard.title.expected-returns"
+                  />
+                </Typography>
+                <ExpectedReturns userPositions={filteredPositions} size="small" isLoading={!hasFetchedUserStrategies} />
```

Comment:
In `ExpectedReturns` and `FinancialOverview`, size=small gives a gap of `6`. For this case, we should use `8`, but it would look like this:
<img width="668" alt="Screenshot 2024-08-16 at 11 32 44 AM" src="https://github.com/user-attachments/assets/aca45142-8a77-442a-a7f0-365651b1e718">
This occurs because figma's max-width is bigger than our App, this is a known difference we already discussed
I think this is 100% acceptable imho, just wanted to give visibility :)

Reply:
God I love how we reuse components